app/views/api/custom_field_definitions/index.json.jbuilder

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/master/LICENSE
+json.array! @custom_field_definitions,
+												partial: '/api/custom_field_definitions/custom_field_definition',
+												as: 'custom_field_definition'
```

Comment:
I find it interesting that rubocop thinks this format looks better.

Reply:
Ya, I hate it. I think the problem is rubocop is really optimized for spaces instead of tabs. Part of me wonders if we should just move to spaces to be like the rest of the Ruby community.

Reply:
I think it would work to manually do:
```suggestion
json.array!(
	@custom_field_definitions,
	partial: '/api/custom_field_definitions/custom_field_definition',
	as: 'custom_field_definition'
)
```
instead of accepting its automatic auto-correct. One thing I noticed about using tabs is that it looks somehow bad when reviewing on Github, but not terrible.

Reply:
Done in latest push!